ecosystem/sep-address-directory-api.md

```diff
@@ -0,0 +1,256 @@
+now-defunct anchors and apps can't be retrieved since their websites don't work
+anymore (stellar.toml is unavailable). The same goes for obsolete exchange
+deposit accounts where people may send funds by mistake.
+
+Custodian accounts that rely on transaction memos make another example of
+entities ignoring ecosystem guidelines. Only a few exchanges adopted quite a
+straightforward [SEP-0029](./sep-0029.md) standard, so the problem of missing
+memos is still relevant. A systematized list of accounts tagged as
+"memo-required" proved to be a convenient solution that helps to check
+memo prerequisites for custodial wallets regardless of their adherence to
+specifications.
+
+While the latter two examples at least partially can be sorted out using
+existing approaches given that entities behind those accounts put some efforts
+into this, there is a category of addresses that will never be mapped that way.
+The problem of people  loosing their money to various frauds is quite crucial –
+law enforcement agencies often can't help in such cases.  Accounts spotted in
+scam schemes or other illicit activity can be identified and blocked only as a
+result of joint community effort. If the community and exchanges
+agree on using Directory API to prevent transfers to/from addresses marked as
+malicious, it will significantly complicate further transfers of stolen funds
+while providing crucial information for compliance checks.
+
+StellarExpert Directory aims to address the problems described above by
+introducing an open REST API interface for the curated list of categorized
+addresses.
+
+## Abstract
+
+The described curated Stellar addresses list provides publicly available
+information about well-known accounts via a set of open REST API endpoints.
+Address Directory API is publicly available for everyone, free of charge.
+
+## Specification
+
+Each directory entry contains the following set of fields:
+
+- `address` - Stellar address of the account
+- `name` - short display name
+- `domain` - domain of the public entity operating the account
+- `tags` - array of account tags (categories)
+
+### REST API specification
+
+#### `GET /directory/tags` - list all available categories
+
+Example:
+
+```
+curl https://api.stellar.expert/explorer/directory/tags
+```
+
+```json
+[
+  {
+    "name": "exchange",
+    "description": "Centralized exchange account"
+  },
+  {
+    "name": "anchor",
+    "description": "Operational account of an anchor"
+  },
+  {
+    "name": "issuer",
+    "description": "Well known asset issuer account"
+  },
+  {
+    "name": "wallet",
+    "description": "Shared account that belongs to a wallet"
+  },
+  {
+    "name": "custodian",
+    "description": "Reserved, custodian account, or cold wallet"
+  },
+  {
+    "name": "malicious",
+    "description": "Account involved in theft/scam/spam/phishing"
+  },
+  {
+    "name": "unsafe",
+    "description": "Obsolete or potentially dangerous account"
+  },
```

Comment:
❓ It's unclear to me what the distinction between malicious and unsafe is practically for a user.

Reply:
"Malicious" are the addresses reported for illicit on-chain activity (scam, spam, funds theft). It implies that wallets should block deposits/withdrawals to such addresses. At the same time, "unsafe" tag refers to organization's accounts that may malfunction for some reason, like no longer used custodial deposit addresses, nonfunctioning anchors, potentially shady ICOs, etc. In this case the wallet can notify the user about potential problem while still allowing transfers to such an address.